### PR TITLE
Feature/winner ui

### PR DESCRIPTION
**Pull request by grace000**
This PR addresses this [story](https://trello.com/c/xpC8B6Ju/29-winning-game-state-board-ui)

**New features**
- Server calculates winning clients
- Server sends client winning coordinates
- Client updates cell components with green color if they correspond to winning coordinates

### CODE REVIEW

File: assets/js/components/Square/index.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 
 const Square = (props) => (
-  <button className="square" onClick={props.onClick}>
+  <button className={props.status} onClick={props.onClick}>
```
Comment (ssunday) [inline assets/js/components/Square/index.js]: is status really apt for this? Classname makes more sense for what the name of thsi props val is

File: assets/js/pages/game.js [hunk 2]
```diff
@@ -38,17 +40,38 @@ class Game extends Component {
...
+    switch(this.state.gameStatus){
+      case "winner":
+        return this.highlightSquare(positionNumber);
+      default: 
+        return "regular-square";
+    }
+  }
+
+  highlightSquare(positionNumber) {
+    let squareClass = "regular-square";
```
Comment (ssunday) [inline assets/js/pages/game.js:53]: const > let, no?

Comment (grace000) [reply assets/js/pages/game.js:53]: The code breaks with const

Comment (ssunday) [reply assets/js/pages/game.js:53]: Oh, yep, yep, it's being redefined.

**Pull request merged**